lib/rig/event_filter.ex

```diff
@@ -270,6 +270,7 @@ defmodule Rig.EventFilter do
 
   """
   @type done_callback :: (() -> nil)
+  @type cloudevent_t :: Cloudevents.t()
```

Comment:
not an issue, just wondering why... I mean `String.t()` is also never aliased to `string_t`...?

Reply:
Because I got this:

`** (CompileError) lib/rig/event_filter.ex:297: cannot invoke remote function Cloudevents.t/0 inside a match`

Reply:
This error refers to the fact that you cannot match on a _type_ on line 297 - this would only work with a _struct_. Not sure why you don't get the same compiler error with the type alias, cause it should really be exactly the same.
However, there is no way to _match_ for multiple structs at once; instead you'd need to use multiple function clauses for `%Cloudevents.Format.V_1_0.Event{}`, `%Cloudevents.Format.V_0_2.Event`, etc. Not sure that's needed here though.